desktop-environment.el

```diff
@@ -182,6 +192,49 @@ portion of the screen."
   (message "New volume value: %s" (desktop-environment-volume-get)))
 
 
+;;; Helper functions - keyboard backlight
+(defun desktop-environment-keyboard-backlight-percent ()
+  "Return the new keyboard backlight value as a % of maximum backlight."
+  (let ((backlight-level (desktop-environment-keyboard-backlight-get)))
+    (if (eq backlight-level 0)
+        (progn (let ((percent "0.0"))
+                 (save-match-data percent)))
+      (progn (let ((percent (*
+                             (/ (* backlight-level 1.0)
+                                (* (desktop-environment-keyboard-backlight-get-max) 1.0))
+                             100)))
+               (save-match-data percent))))))
+
+(defun desktop-environment-keyboard-backlight-get ()
+  "Return a string representing keyboard backlight current level."
+  (let ((output (dbus-call-method :system
+                                   "org.freedesktop.UPower"
+                                   "/org/freedesktop/UPower/KbdBacklight"
+                                   "org.freedesktop.UPower.KbdBacklight"
+                                   "GetBrightness")))
+    (save-match-data output)))
```

Comment:
the `let` and `save-match-data` are useless here as well.

Reply:
See comment above. Removed.